packages/protocol/contracts/mock/MockAirnodeRrpClient.sol

```diff
@@ -91,4 +91,30 @@ contract MockAirnodeRrpClient is AirnodeRrpClient {
     delete incomingFulfillments[requestId];
     emit RequestFulfilled(requestId, statusCode, data);
   }
+
+  /// @notice A method to be called back by the respective method at
+  /// AirnodeRrp.sol for testing fulfillment failure
+  /// @param /* requestId */ Request ID
+  /// @param /* statusCode */ Status code returned by the Airnode
+  /// @param /* data */ Data returned by the Airnode
+  function fulfillAlwaysReverts(
+    bytes32, /* requestId */
+    uint256, /* statusCode */
+    bytes calldata /* data */
+  ) external view onlyAirnodeRrp() {
+    revert("Forcing function to revert");
+  }
+
+  /// @notice A method to be called back by the respective method at
+  /// AirnodeRrp.sol for testing fulfillment running out of gas
+  /// @param /* requestId */ Request ID
+  /// @param /* statusCode */ Status code returned by the Airnode
+  /// @param /* data */ Data returned by the Airnode
+  function fulfillAlwaysRunsOutOfGas(
+    bytes32, /* requestId */
+    uint256, /* statusCode */
+    bytes calldata /* data */
+  ) external view onlyAirnodeRrp() {
+    while (true) {}
```

Comment:
Good catch, I couldn't find a quick workaround to disable just this one instance of the eslint warning so maybe we could hack it by adding a `continue` statement inside the while loop:

  ```
function fulfillAlwaysRunsOutOfGas(
    bytes32, /* requestId */
    uint256, /* statusCode */
    bytes calldata /* data */
  ) external view onlyAirnodeRrp() {
    while (true) {
      /* this loop runs until gas limit is reached */
      continue;
    }
  }
```

Reply:
Disregard previous comment. The warning was actually coming from solhint and not eslint. This is how to prevent getting `no- empty-blocks` warnings: 

```
  /// @notice A method to be called back by the respective method at
  /// AirnodeRrp.sol for testing fulfillment running out of gas
  /// @param /* requestId */ Request ID
  /// @param /* statusCode */ Status code returned by the Airnode
  /// @param /* data */ Data returned by the Airnode
  function fulfillAlwaysRunsOutOfGas(
    bytes32, /* requestId */
    uint256, /* statusCode */
    bytes calldata /* data */
  ) external view onlyAirnodeRrp() {
    while (true) {}
    // solhint-disable-previous-line no-empty-blocks
  }
```